openapi/src/json-schemas/api/resources/answer-resource.json

```diff
@@ -0,0 +1,14 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "title": "Answer resource",
+    "allOf": [
+        {"$ref": "../json-api-base-resource.json"},
+        {
+            "properties": {
+                "type": {
+                    "const": "answers"
+                }
```

Comment:
I know that `attributes` is defined in `json-api-base-resource.json`, but should this have a specific attributes definition?

Reply:
It could, I'm not sure of the value, certainly with OpenAPI. It wouldn't be able to check that the keys are valid question ids as there are is no pattern support, and the value of the keys can be any JSON ,which is what the "{}" schema allows

Reply:
Any thoughts on what could be added?

Reply:
I can see that the value can be any JSON. was just wondering if it was worth tightening it at all at this point

Reply:
I was thinking that there was a discernible shape that would not require `patternProperties` to be defined here. But that is exactly what it is. Had arrays on the brain.

Reply:
That's what I mean, I'm not too sure on what else could be added, any thoughts?

Reply:
no! I was thinking that the answers was an array for whatever reason. ignore me